project/plugins.sbt

```diff
@@ -1,10 +1,13 @@
 
 resolvers += Resolver.sonatypeRepo("public")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-pgp" % "0.8.1")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-M15")
```

Comment:
Adding this to Reactors previously broke a multi-project mecha-based build (possibly because all the projects required the same sbt plugin).